### DISCUSSION TITLE
Unable to finetune rows/columns

### DISCUSSION THREAD

**Discussion by rachittshah45** [jsvine/pdfplumber#582] [Q&A]
Hey community!

I have tried to remove tables from an image based pdf, but whenever I try to remove data from the tables, the rows get clubbed together or the first character from each row overflows into the row above it. 

What could be possible solutions to fixing the issue?

Thanks in advance.

**Comment by jsvine** [comment]
Hi @rachittshah45, and thanks for your interest in this library. Unfortunately, it's a bit hard to provide advice or suggestions without seeing the PDF or code you're using. Could you provide those things?
